### ISSUE TITLE
Wagtail: image and text 25/75 requires alt image tag twice

### ISSUE THREAD

**Issue by anselmbradford** [cfpb/cfgov-refresh#2853]
## Current behavior

- When an image is uploaded in wagtail an alt text field is filled in for the image.
- When an image from the wagtail image library is used for an image and text 25/75 component, the image has an alt text field also.

## Expected behavior

- After the alt image text field is filled in for an uploaded image that alt text should just be used where ever the image is used.

## Steps to replicate behavior

1. Upload an image to the wagtail image library. Give it alt text.
2. Add an Image Text 25 75 component to a page.
3. Add an image from the library. 
4. See that you have to enter the alt text in again.

## Screenshots

__Current__

![screen shot 2017-04-18 at 6 36 40 pm](https://cloud.githubusercontent.com/assets/704760/25155992/09f09f16-2466-11e7-8779-46abc0eb5c02.png)

![screen shot 2017-04-18 at 6 36 49 pm](https://cloud.githubusercontent.com/assets/704760/25155993/09f5d558-2466-11e7-9dbd-b0ffa1274385.png)

**Comment by Scotchester** [comment]
There's a related bug where, if no alt text is entered on an image in a 50/50 Image and Text module and it's set to use a 16:9 image, the image will not be rendered.

In both cases, fixing the issue should be done in such a way as to preserve the ability to render an image with no alt text (either as `<img alt="">` or as a CSS `background-image`), for cases in which we wouldn't want a screenreader to read anything for that image.

**Comment by jimmynotjim** [comment]
I remember now we had set it up to use an `img` if the `alt` text existed and a `background-image` if it didn't. We must have missed something along the way though that caused the 50/50 to break.

**Comment by anselmbradford** [comment]
25/75 component has been removed (see https://github.com/cfpb/cfgov-refresh/pull/5624 and https://github.com/cfpb/cfgov-refresh/pull/5906)
